INSTALL.md

```diff
@@ -0,0 +1,47 @@
+Building with cmake
+-------------------
+
+1. Download and install [vagrant](https://www.vagrantup.com).
+
+2. Download and install [VirtalBox](https://www.virtualbox.org).
+
+3. Obtain and launch the RadiaSoft development environment:
+
+```bash
+curl radia.run | bash -s vagrant-sirepo-dev
```

Comment:
The order isn't right. Perhaps just link to https://github.com/radiasoft/sirepo/wiki/Development#mac-install